.gitignore

```diff
@@ -34,3 +34,4 @@ queue_listeners/config/env.yml
 !.elasticbeanstalk/*.global.yml
 node_modules
 public/system
+config/unicorn.rb
```

Comment:
I was using it temporarily in development. Having unicorn in production is a little more involved - something for another PR.